crates/router/src/connector/aci/transformers.rs

```diff
@@ -209,9 +237,25 @@ impl
                     customer_email: None,
                 }))
             }
-            _ => Err(errors::ConnectorError::NotImplemented(
-                "Payment method".to_string(),
-            ))?,
+            api_models::payments::BankRedirectData::Bizum { .. }
+            | api_models::payments::BankRedirectData::Blik { .. }
+            | api_models::payments::BankRedirectData::Eps { .. }
+            | api_models::payments::BankRedirectData::Giropay { .. }
+            | api_models::payments::BankRedirectData::Ideal { .. }
+            | api_models::payments::BankRedirectData::Interac { .. }
```

Comment:
```suggestion
```
Please remove these match arms as these are already handled

Reply:
We could see the change, can you please remove these, this is the reason clippy check is failing